koku/api/provider/test/tests_serializers.py

```diff
@@ -126,7 +131,7 @@ def test_create_all_providers(self):
     def test_create_provider_fails_user(self):
         """Test creating a provider fails with no user."""
         provider = {'name': 'test_provider',
-                    'type': Provider.PROVIDER_AWS,
+                    'type': Provider.PROVIDER_AWS.lower(),
```

Comment:
Why did all of these have to get set to `lower()`?

Reply:
These were set to `.lower()` because I was lowercasing the type of the incoming request in the view (so the serializer expects it to be lowercase). I am open to changing this if you would like!